### DISCUSSION TITLE
wrong function provided in a simple task calculateAge

### DISCUSSION THREAD

**Discussion by Mhossni94** [community/community#14828] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/39713199/163082790-10ed66d2-9f68-4e76-a563-2214d56b039a.png)

**Comment by FortMyersBrewing** [comment, marked as answer]
You didn't give copilot anything to work with there, it gave you exactly what you asked for.."calculateAge".. based on what? It could have responded with '2015 - birthyear' and technically it answered your question. Attached is what my CoPilot gave me (shown on the right) when I asked it a similar question:
<img width="1434" alt="Screen Shot 2022-04-12 at 10 31 49 PM" src="https://user-images.githubusercontent.com/10420504/163088168-45ad89e0-3fd3-404c-b99e-4f49d9624008.png">

**Reply by Mhossni94** [reply]
Thanks for your feedback, I will try to be more descriptive :D
